functional/server_test.go

```diff
@@ -96,7 +122,7 @@ func waitForReloadConfig(cluster platform.Cluster, m0 platform.Member) (err erro
 			// "journalctl -u fleet | grep \"Reloading configuration\"" is racy
 			// in a subtle way, so that it sometimes fails only on semaphoreci.
 			// - dpark 20160408
-			stdout, _ := cluster.MemberCommand(m0, "journalctl _PID=$(pidof fleetd)")
+			stdout, _ := cluster.MemberCommand(m0, "sudo", "journalctl --priority=info _PID=$(pidof fleetd)")
```

Comment:
What does this change do? I don't think it's explained anywhere -- or did I miss it?...